javascript-local-storage/data.js

```diff
@@ -0,0 +1,12 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+if (previousTodosJSON !== null) {
+  todos = [];
```

Comment:
if previous todos is not null, you want to assign it to the `todos` variable. This is how you take the data from local storage and use it in the app.